### ISSUE TITLE
revenue model is '' for everyone

### ISSUE THREAD

**Issue by whit537** [gratipay/gratipay.com#3482]
cf. #3479

**Comment by whit537** [comment]
Actually, I'm going to reopen this until we're able to communicate with all of the people whose answer to the revenue question we threw away. :-(

**Comment by whit537** [comment]
And of course that won't do much good until we have https://github.com/gratipay/gratipay.com/issues/3487. :-(

**Comment by whit537** [comment]
+1 from @toy at https://github.com/gratipay/review/issues/3#issuecomment-118190649.

**Comment by mattbk** [comment]
All teams left in the [queue](https://github.com/gratipay/review/issues) have the revenue section filled in.

**Comment by whit537** [comment]
I think we should keep this open until we repair the first batch of teams that *don't* have this field filled in.

**Comment by whit537** [comment]
That amounts to apologetically emailing all of them to fill in the field again, which would make more sense once we can edit team pages more easily (#3545).

**Comment by mattbk** [comment]
In the meantime, we need a list of teams that have this field blank.

**Comment by whit537** [comment]
@mattbk You should be able to track that down on https://gratipay.com/explore/teams/. It's sorted by date created, descending, so start at the bottom and work your way up.

**Comment by mattbk** [comment]
Still blank for
- https://gratipay.com/introtopython/ @ehmatthes
- https://gratipay.com/duo/ @mixolidia
- https://gratipay.com/sudo-room/
- https://gratipay.com/nuvola-player/
- https://gratipay.com/cyberpipe/ @kiberpipa
- https://gratipay.com/mojolicious/ @kraih
- https://gratipay.com/sublimelinter/ @grotewold
- https://gratipay.com/pageres/ @sindresorhus
- https://gratipay.com/kivy/
- https://gratipay.com/citizens-cooperative/
- https://gratipay.com/catapultpgh/
- https://gratipay.com/sudomesh/ @sudomesh
- https://gratipay.com/perl-maven/
- https://gratipay.com/exercism/ @kytrinyx
- https://gratipay.com/the-mountain-spirit/ @heidigardner
- https://gratipay.com/weblate/ @nijel
- https://gratipay.com/devdocs/ @thibaut
- https://gratipay.com/jshttp/ @dougwilson
- https://gratipay.com/pillow/ @aclark4life

**Comment by whit537** [comment]
!m @mattbk

**Comment by mixolidia** [comment]
I don't know how to update that section. How do I update that section?

**Comment by whit537** [comment]
You can't yet, @mixolidia, sorry. :-(

I/we can update it manually for you, or you can wait for https://github.com/gratipay/gratipay.com/issues/3545.

**Comment by mixolidia** [comment]
It's okay. I can wait. I wasn't sure if I'd forgotten to do something. Thanks!

**Comment by dougwilson** [comment]
Reading this issue, I am listed, bit it is not clear if there is something I need to do, and if so, how to go about it.

**Comment by mattbk** [comment]
@dougwilson You don't need to do anything at this time, but once #3545 is done, we'll be sending you an email.

**Comment by whit537** [comment]
Those watching this ticket may be interested in the discussion we're having about [revising the team application/page](https://github.com/gratipay/gratipay.com/issues/3677).

**Comment by whit537** [comment]
This is moot with #3679. We no longer ask for revenue model.
